spec.html

```diff
@@ -12003,6 +12004,9 @@ <h1>Runtime Semantics: GetTemplateObject ( _templateLiteral_ )</h1>
         <emu-note>
           <p>Future editions of this specification may define additional non-enumerable properties of template objects.</p>
         </emu-note>
+        <emu-note>
+          The [[TemplateLiteral]] internal slot is created on template objects in order to enable APIs in embedding environments to check whether an object is a template object. The value of the slot is typically *undefined*.
```

Comment:
I was also confused; I thought `[[TemplateLiteral]]` was intended to hold some reference to something. Are there other places where we are using an internal slot for a branch check without ever storing anything in that slot?

Reply:
Errors have an `[[ErrorData]]` internal slot, which is always `undefined`. Similarly, `Intl.DateTime` objects have a `[[InitializedDateTimeFormat]]`internal slot, always `undefined`. Just trying to follow existing conventions--I'm happy to phrase this branding in some other way if you have a recommendation.

Reply:
Ah, right - `[[ErrorData]]`.

The naming convention for these things could probably be improved a bit for clarity, but with the note it looks good to me.

👍

Reply:
`[[ErrorData]]`, at least, is definitely used in the stage 1 stacks proposal.